src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOauthClientCredentialClientCommand.php

```diff
@@ -542,4 +542,10 @@ public function getTypeOfService(): array
         // not implemented for oauth_ccc entities
         return [];
     }
+
+    public function isPublicInDashboard(): ?bool
+    {
+        // This coin is not used for CCC entities
+        return null;
```

Comment:
Why not return false?

Reply:
When a coin is not set in Manage, we consider it to be `null` in SP Dashbaord.
When a boolean coin is present (like `coin:ss:idp_visible_only`) the other values can be true or false.

When a certain entity type does not support a certain coin. I feel returning `null` by default works out the best.